numpy/core/tests/test_indexerrors.py

```diff
@@ -1,5 +1,8 @@
 import numpy as np
-from numpy.testing import assert_raises
+from numpy.testing import (
```

Comment:
I think you can just do `from numpy.testing import assert_raises, assert_raises_regex`, it should still be in the limit.